src/components/DetailsPages/Misc/InfoTabs/InitParamsTab.tsx

```diff
@@ -25,7 +25,13 @@ const InitParamsTab: React.FC<IProps> = ({ initParams }) => {
             <td>{index}</td>
             <td>{param.vname}</td>
             <td>{param.type}</td>
-            <td>{Array.isArray(param.value)
+            <td>{param.value !== undefined ?
```

Comment:
This whole block is just unreadable given the number of ternary operators.
Given the (bad) indentation, it's also unclear what some of those operators relate to, e.g.:
`param.value !== undefined ? ... :Array.isArray(param.value) ? param.value.map((x, index) => (`
              
Is it possible to fix this? ideally move some of those blocks into separate functions?